### ISSUE TITLE
[HW7] meaning of "specification"

### ISSUE THREAD

**Issue by KimNaRyeong** [swtv-kaist/cs458-fall22#32]
Should I write it down like the picture below? (Should I write down everything from benchmark sort to formula?)
![image](https://user-images.githubusercontent.com/93383680/206760173-c6e65dfa-9fbb-4ceb-826b-e31684254fa1.png)

Also in Problem1.2, should I write the SSA form of assert statement to be written in the Problem 1.1?

**Comment by moonzoo** [comment]
> Should I write it down like the picture below? (Should I write down everything from benchmark sort to formula?)

Yes, you can write down your HW solution similar to the slide.  

> Also in Problem1.2, should I write the SSA form of assert statement to be written in the Problem 1.1?

In problem 1.2, you don't need to write down an assert statement in a SSA form. 
But anyway, you need it for problem 1.3 and 1.4.  See https://github.com/swtv-kaist/cs458-fall22/issues/24
